### PR TITLE
Fixed some glitches

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Some minor typos that broke the markdown parsing rendering

### CODE REVIEW

File: UPGRADE-1.7.md [hunk 1]
```diff
@@ -38,30 +38,30 @@
     * Download it from the website [PIM community standard](http://www.akeneo.com/download/) and extract:
```
Comment (pierallard) [inline UPGRADE-1.7.md]: Please remove the line just under (bad copy paste)

Comment (mickaelandrieu) [inline UPGRADE-1.7.md]:
```diff
_ 1. Download and extract the latest standard archive,
+ 1. Download it from the website [PIM community standard](http://www.akeneo.com/download/) and extract:
```

Comment (pierallard) [inline UPGRADE-1.7.md]: yep

**Comment by mickaelandrieu** [comment]
Thanks for reviews guys :)

**Pull request merged**